File: Backend/WriteAPI/Common/DTO/folders/FullFolder.cs
using Common.DatabaseModels.models;
using Common.DatabaseModels.models.Folders;
using Common.DatabaseModels.models.Systems;
using Common.DTO.app;
using System;
using System.Collections.Generic;
using System.Text;

namespace Common.DTO.folders
{
    public class FullFolder
    {
        public Guid Id { get; set; }
        public string Title { set; get; }
        public string Color { set; get; }
        public FolderTypeENUM FolderTypeId { set; get; }
        public RefTypeENUM RefTypeId { set; get; }
        public DateTimeOffset DeletedAt { set; get; }
        public DateTimeOffset UpdatedAt { set; get; }
        public DateTimeOffset CreatedAt { set; get; }
        public List<NotePreviewInFolder> PreviewNotes { set; get; }

    }
}

File: Client/src/app/content/folders/models/FullFolder.ts
import { FolderTypeENUM } from 'src/app/shared/enums/FolderTypesEnum';
import { RefTypeENUM } from 'src/app/shared/enums/refTypeEnum';

export interface FullFolder {
  id: string;
  title: string;
  color: string;
  refTypeId: RefTypeENUM;
  folderTypeId: FolderTypeENUM;
}
